components/TimeBarChart.vue

```diff
@@ -75,15 +110,15 @@ export default {
       if (this.dataKind === 'transition') {
         return {
           lText: `${this.chartData.slice(-1)[0].transition.toLocaleString()}`,
-          sText: `実績値（前日比：${this.displayTransitionRatio} ${this.unit}）`,
+          sText: `${this.$t('実績値')}（${this.$t('前日比')}：${this.displayTransitionRatio} ${this.unit}）`,
           unit: this.unit
         }
       }
       return {
         lText: this.chartData[
           this.chartData.length - 1
         ].cumulative.toLocaleString(),
-        sText: `${this.chartData.slice(-1)[0].label} 累計値（前日比：${
+        sText: `${this.chartData.slice(-1)[0].label} ${this.$t('累計値')}（${this.$t('前日比')}：${
```

Comment:
こちらも、全角コロンの部分を半角コロン+スペースに修正お願いいたします
```suggestion
        sText: `${this.chartData.slice(-1)[0].label} ${this.$t('累計値')}（${this.$t('前日比')}: ${
```

Reply:
こちらも対応しました